shared/tasks/assets.mjs

```diff
@@ -84,7 +86,6 @@ export async function write (filePath, result) {
  * @typedef {object} AssetFileOptions
  * @property {(file: import('path').ParsedPath) => string} [filePath] - File path formatter
  * @property {(contents?: string) => Promise<string>} [fileContents] - File contents formatter
- * @property {string[]} [ignore] - File path patterns to ignore
```

Comment:
We had to carefully ignore the published **package.json** and **README.md** in the past

We haven't used this option since merging:

* https://github.com/alphagov/govuk-frontend/pull/3498